kickstart/decide-presenters.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Decide who will present the kickstart presentation
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Kickstart
+
+description: >-
+    The presentation needs to be given at each venue, so will be presented by
+    multiple people. It's recommended the person who creates the presentation be one of the people to
+    present.
+
+dependencies:
+    - kickstart/write-presentation
```

Comment:
Is this strictly a dependency? I realise that https://github.com/srobo/recurring-tasks/issues/25 complicates cross linking, but I think we should probably still aim to keep dependencies as just that.